com.github.opentyrian.OpenTyrian.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: com.github.opentyrian.OpenTyrian
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --device=all
+  - --share=network
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Pulseaudio access
+  - --socket=pulseaudio
+  # OpenGL access
+  - --device=dri
+  #SDL needs to be able to talk to UPower
+  - --system-talk-name=org.freedesktop.UPower
+  #Allow SDL to inhibit the screensaver
+  - --talk-name=org.freedesktop.ScreenSaver
+command: opentyrian
+rename-appdata-file: opentyrian.appdata.xml
+rename-desktop-file: opentyrian.desktop
+rename-icon: opentyrian
+copy-icon: true
+modules:
+  - name: opentyrian
+    buildsystem: simple
+    build-commands:
+      - make install DESTDIR=${FLATPAK_DEST}/ prefix= gamesdir=extra/
+    post-install:
+      - install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra
+      - install -Dm644 opentyrian.appdata.xml -t ${FLATPAK_DEST}/share/metainfo/
+      - install -Dm644 linux/opentyrian.desktop -t ${FLATPAK_DEST}/share/applications/
+      - install -Dm644 linux/icons/tyrian-128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/opentyrian.png
+      - install -Dm644 linux/icons/tyrian-48.png ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/opentyrian.png
+      - install -Dm644 linux/icons/tyrian-32.png ${FLATPAK_DEST}/share/icons/hicolor/32x32/apps/opentyrian.png
+      - install -Dm644 linux/icons/tyrian-24.png ${FLATPAK_DEST}/share/icons/hicolor/24x24/apps/opentyrian.png
+      - install -Dm644 linux/icons/tyrian-22.png ${FLATPAK_DEST}/share/icons/hicolor/22x22/apps/opentyrian.png
+    sources:
+      - type: git
+        url: https://github.com/opentyrian/opentyrian
+        tag: v2.1.20220318
+        commit: 8f205e6b12cb9c34905fa960416b6f661bf38ac3
+      - type: file
+        path: opentyrian.appdata.xml
+      - type: extra-data
+        filename: tyrian21.zip
+        url: https://camanis.net/tyrian/tyrian21.zip
+        sha256: 7790d09a2a3addcd33c66ef063d5900eb81cc9c342f4807eb8356364dd1d9277
+        size: 4754048
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - unzip -q tyrian21.zip
+          - rm tyrian21.zip
+          - mkdir -p ${FLATPAK_DEST}/share/tyrian
+          - mv ${FLATPAK_DEST}/extra/tyrian21 ${FLATPAK_DEST}/share/tyrian
```

Comment:
this won't work.
1. `${FLATPAK_DEST}` isn't defined anymore at this stage
2. `/app/share` IS read-only.

The solution as commented elsewhere is to make sure `gamesdir` is set to `${FLATPAK_DEST}/extra` at build time.

Reply:
removed